src/Package/Impl/Repl/Shiny/RunShinyAppCommand.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.R.Components.InteractiveWorkflow;
+using Microsoft.R.Host.Client;
+using Microsoft.VisualStudio.R.Package.Commands;
+using Microsoft.VisualStudio.R.Packages.R;
+
+namespace Microsoft.VisualStudio.R.Package.Repl.Shiny {
+    internal sealed class RunShinyAppCommand : PackageCommand {
+        private readonly IRInteractiveWorkflow _interactiveWorkflow;
+
+        public static Task RunningTask { get; private set; }
```

Comment:
So I can find it from StopShinyAppCommand. Alternative is a bit more complicated: locate menu command service or UI shell and ask for the command status .